src/screens/NewAnalytics/PaymentAnalytics/PaymentsProcessed/PaymentsProcessedTypes.res

```diff
@@ -8,3 +8,5 @@ type paymentsProcessedCols =
   | Count
   | Amount
   | TimeBucket
+
+type categories = [#time_bucket]
```

Comment:
`TimeBucket` is already there as a variant u can use the same only, if it need to be converted to string maybe use a mapper function

Reply:
okay